packages/KStepper/stepper-icons/KCompletedState.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <svg
+    :width="size"
+    :height="size"
+    viewBox="0 0 24 24"
+    fill="none"
+    data-testid="k-step-completed"
+  >
+    <title>{{ title }}</title>
+    <circle
+      :fill="color"
+      cx="12"
+      cy="12"
+      r="12"
+    />
+    <path
+      :fill="color"
+      d="M16 9L10 15L7 12.0001"
+    />
+    <path
+      :stroke="backgroundColor"
+      d="M16 9L10 15L7 12.0001"
+      stroke-width="2.25"
+      stroke-linecap="round"
+      stroke-linejoin="round"
+    />
+  </svg>
+</template>
+
+<script>
+export default {
+  name: 'KCompletedState',
```

Comment:
For these `K*State` components within the `stepper-icons` folder, would it make sense to rename them to `KCompletedIcon`, `KDefaultIcon`, etc. instead?